### ISSUE TITLE
When I create an action, a gray panel hides all the actions to be defined.

### ISSUE THREAD

**Issue by Melcreatie** [FlutterFlow/flutterflow-issues#1454]
### Has your issue been reported?

- [X] I have searched the existing issues and confirm it has not been reported.
- [X] I give permission for members of the FlutterFlow team to access and test my project for the sole purpose of investigating this issue.

### Current Behavior

When I create an action, a gray panel hides all the actions to be defined.

### Expected Behavior

View and define actions

### Steps to Reproduce

1. I create a page 
2. I click on a button to add an action 
3. A gray panel appears, hiding all the actions to be defined 
4. Bug Issues Error --> Action configuration

### Reproducible from Blank

- [X] The steps to reproduce above start from a blank project.

### Bug Report Code (Required)

IT4KiMjfsI52ocMA+KX5bcFr/To5KVgjauYslutub0k3JYjpG7l/TMnOVBJuOLC0T1FcPGagjT4f0PPemvGTU+4CHAWcGLph1aVUSQmXWka6R7qgBYeOWG5/AptYFXKx36uRsxNAH7diSVIZ7WCqGOeiZHbrGZzIOjEeQ8ilC5CK2SrDX1iXc2URm05KZDPz

### Context

I wanted to create an action

### Visual documentation

![image](https://github.com/FlutterFlow/flutterflow-issues/assets/28703086/aeb803fd-a82c-4449-87c0-9d6947c92ad0)

### Environment

```markdown
- FlutterFlow version: Version 1.1.101 (1.1.101)
- Platform: MacOS Desktop
- Browser name and version: FlutterFlow App beta
- Operating system and version affected: macOS Big Sur v11.7
```

**Comment by hariprasadms** [comment]
Hi @Melcreatie - Thanks for submitting the issue. I just tested the above steps and was not able to reproduce the grey page. Can you please add bit more details about the issue. Example, screenshots and a quick video and etc. This could be a project specific issue. I suggest test this more than once and also in different projects to see the consistency in behaviour. Please update these details in the issue. Thanks

**Comment by hariprasadms** [comment]
Hi, Thanks for the details. 
Dev note - It is project and Mac OS specific issue.

**Comment by wenkaifan0720** [comment]
Hi @Melcreatie. This should be fixed in our latest Beta v1.1.102. Thanks.

**Comment by sabatage3** [comment]
I have the exact same issue in macOS, it's September 22 already, any idea when this next beta will be released?
